index.js

```diff
@@ -28,58 +29,107 @@ const saveAs = (uri, filename) => {
 /**
  * @param  {React.RefObject} node
  * @param  {string} fileName
- * @param  {string} backgroundColor
  * @param  {string} type
- * @param  {object} options=null
+ * @param  {object} html2CanvasOptions={}
  */
-const exportComponent = (node, fileName, backgroundColor, type, options) => {
+
+const getPDF = (canvas, {x, y, orientation}) => {
+    const width = x || canvas.width
```

Comment:
This x, y I believe is useful in the alignment when we call `pdf.addImage(canvas.toDataURL(fileType.PNG, 1.0), 'PNG', x, y);`

https://artskydj.github.io/jsPDF/docs/module-addImage.html

Reply:
i read through the docs and made additional changes. let me know what you think